### PR TITLE
Knowledge questions about common tasks

### PR DESCRIPTION
**Pull request by JamesHovious**

### CODE REVIEW

File: knowledge.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 2. What kind of attack is ARP Spoofing considered and how could you leverage it on a penetration test? *@jstnkndy*
 3. Explain what NBNS poisoning is and how it can be leveraged on a penetration test. *@jstnkndy*
 12. Answer true or false and explain your answer: two-factor authentication protects against session hijacking. *@jstnkndy*
+13. How would you avoid anti virus?
```
Comment (WebBreacher) [inline knowledge.md]: Can you please move this to the open.md file since there are multiple methods of doing it?

File: knowledge.md [hunk 2]
```diff
@@ -7,6 +7,10 @@
 2. What kind of attack is ARP Spoofing considered and how could you leverage it on a penetration test? *@jstnkndy*
 3. Explain what NBNS poisoning is and how it can be leveraged on a penetration test. *@jstnkndy*
 12. Answer true or false and explain your answer: two-factor authentication protects against session hijacking. *@jstnkndy*
+13. How would you avoid anti virus?
+14. How would you target a database that you know lies behing a jump server with an unkown IP address?
```
Comment (WebBreacher) [inline knowledge.md]: Can you please move this to the open.md file since there are multiple methods of doing it?

File: knowledge.md [hunk 3]
```diff
@@ -7,6 +7,10 @@
 2. What kind of attack is ARP Spoofing considered and how could you leverage it on a penetration test? *@jstnkndy*
 3. Explain what NBNS poisoning is and how it can be leveraged on a penetration test. *@jstnkndy*
 12. Answer true or false and explain your answer: two-factor authentication protects against session hijacking. *@jstnkndy*
+13. How would you avoid anti virus?
+14. How would you target a database that you know lies behing a jump server with an unkown IP address?
+15. What are commonly vulnerable ports/services/applications? 
+16. You have unprivileged Windows credentials, how can you elevate your privileges?
```
Comment (WebBreacher) [inline knowledge.md]: Can you please move this to the open.md file since there are multiple methods of doing it?

File: knowledge.md [hunk 4]
```diff
@@ -20,6 +24,8 @@
 ### Web Application ###
 4. Describe what SQL Injection is and how you would test for it? *@jstnkndy*
 5. What about Blind SQL Injection and how is it different from other kinds? *@jstnkndy*
+6. How can SQL Injection lead to remote code execution?
```
Comment (WebBreacher) [inline knowledge.md]: Can you please move this to the open.md file since there are multiple methods of doing it?

**Comment by JamesHovious** [comment]
Made requested changes, sorry for the late update I never saw the notification from GitHub

**Pull request merged**